docs/binary.md

```diff
@@ -36,11 +32,10 @@ Binary Number :: 11001
 ---
 
 ## Binary Counting
-<<<<<<< Updated upstream
 
-<h2>How do we count using binary?</h2>
-=======
->>>>>>> Stashed changes
+## How do we count using binary?
```

Comment:
Another small attention to detail, it is in best practices to have a smaller dimension heading under a given heading instead of having the heading and sub-heading as the same dimension. For example, < h2 > level headings should not have sub-headings of < h2 > level, instead they should be a dimension (or more) smaller, like < h3 > or < h4 >

Over here, _How do we count using binary?_ is a subheading to _Binary Counting_ and thus should not be at par with it's parent-heading level. I would suggest keep the subheadings at < h3 > i.e. ### level and the headings at < h2 > i.e. ## level